run_dir/design/instrument_logs.html

```diff
@@ -13,29 +13,36 @@
 <h1>Instrument Logs</h1>
 <div id="logs_date" style="margin-bottom:15px;">
     <p>The default view shows 1 week worth of logs. You can customise the range here :</p>
-    <form class="form-inline" id="logs_date_form">
-        Select logs from
-        <div class='input-group input-group-sm date' id='datepick1'>
-            <input id="inp_date_1" type='text' class="form-control" /><span class="input-group-addon"><span class="glyphicon glyphicon-calendar"></span></span>
+    <form class="row row-cols-lg-auto" id="logs_date_form">
+        <div class='col-12'>
+          Select logs from
+        </div>
+        <div class='col-12'>
+          <div class='input-group input-group-sm date' id='datepick1'>
```

Comment:
I have some issue here which is related to how the top navbar seems to work now, see attached image. 
<img width="340" alt="Screenshot 2020-12-18 at 07 31 32" src="https://user-images.githubusercontent.com/1250075/102582572-2ec06480-4103-11eb-99e3-90be9d37240d.png">

Not sure how to solve it, is there a way to make navbar non-overlapping with the body?

Reply:
The fix is up on stage!